src/otx/config/data/openvino.yaml

```diff
@@ -2,6 +2,7 @@ defaults:
   - default
 
 image_color_channel: BGR
+stack_images: False
```

Comment:
I cannot seem to find any description in any of the docstrings for `stack_images`. Would it be possible to add some description in the relevant docstring to tell the user what it is and what it does?

Reply:
I added a docstring for the OTXDataset and a description for the parameters from the data config.

Reply:
ah sorry, you had to go through the other params too :) Thanks for doing that though!